examples/rust/.test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -ex
+cargo --version
+rustc --version
+
+echo "$RUSTFLAGS" | grep -- "--jobs 1"
```

Comment:
Is there a way to build a simple rust/cargo application inside the test in order to test whether these rustflags work end-to-end?

Grepping the values doesn't say much about the usecase that it is trying to support.

Reply:
Mostly rust flags is used to inform the rust compiler of other libraries to link against. I don't usually develop on macOS, but I tried adding some dependencies that use native libraries on macOS (notify crate) and then compiling the project but it seemed to work without these flags set. It could be that you have to use something specific.

In that case, perhaps these libraries shouldn't even be linked by default since many rust programs won't need them at all even if they're being developed on macOS.

Reply:
I added an example rust app and compile / run it in the test. It doesn't cover the linking with the macos flags though.